test/windows/UnitTests.cpp

```diff
@@ -2234,7 +2267,8 @@ Error code: Wsl/InstallDistro/WSL_E_DISTRO_NOT_FOUND
         // Keys that are created by the optional component and the service.
         const std::vector<LPCWSTR> inboxKeys{
             L"SOFTWARE\\Classes\\CLSID\\{B2B4A4D1-2754-4140-A2EB-9A76D9D7CDC6}",
-            L"SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Explorer\\Desktop\\NameSpace\\{B2B4A4D1-2754-4140-A2EB-9A76D9D7CDC6}",
+            L"SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Explorer\\Desktop\\NameSpace\\{B2B4A4D1-2754-4140-A2EB-"
+            L"9A76D9D7CDC6}",

```

Comment:
Hmm why is this getting wrapped? Do you have a different version of clang format?

Reply:
Potentially. We use the version that comes with VS now, so it gets updated automatically